openreview/conference/builder.py

```diff
@@ -2391,57 +2391,62 @@ def get_readers(self, conference, number, review_signature):
 
 class CommentStage(object):
 
+    class Readers(Enum):
+        EVERYONE = 0
+        SENIOR_AREA_CHAIRS = 1
```

Comment:
```suggestion
        SENIOR_AREA_CHAIRS_ASSIGNED = 1
```

Let's use assigned so it doesn't get confused with the general senior area chairs group